starmark-integration/tests/cloudflare-env-adapter.test.ts

```
@@ -0,0 +1,138 @@
+import { describe, test, expect, beforeEach } from 'vitest';
+import { CloudflareEnvAdapter, type CloudflareContext } from '../src/storage/CloudflareEnvAdapter';
+
+describe('CloudflareEnvAdapter', () => {
+  let adapter: CloudflareEnvAdapter;
+
+  beforeEach(() => {
+    adapter = new CloudflareEnvAdapter();
+    // Reset process.env for each test
+    delete process.env.TEST_VAR;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)